api/db_queries.py

```diff
@@ -231,8 +232,14 @@
 """
 
 QUERY_PARALELLS_FOR_MIDDLE_TEXT = """
+let parallel_ids = (
+    FOR segment in segments
+        FILTER segment._key == @segmentnr
```

Comment:
I think I will leave it for now because these things happen so often that it is best to do it consistently throughout in one go, also with the jsons updated with snake_case.